addons/website/static/src/snippets/s_instagram_page/000.js

```diff
@@ -73,16 +73,6 @@ const InstagramPage = publicWidget.Widget.extend({
      */
     _onMessage(ev) {
         const iframeEl = this.el.querySelector(".o_instagram_container iframe");
-        if (!iframeEl) {
-            // TODO: fix this case. We should never end up here. It happens when
-            // - Drop Instagram snippet
-            // - Undo
-            // - Drop a new Instagram snippet
-            // => The listener of the first one is still active because the
-            // public widget has not been destroyed.
-            window.removeEventListener("message", this.__onMessage);
-            return;
-        }
```

Comment:
I take the opportunity to remove this.

Note that if you want to check that my test is failing without the fix, you will have to apply the changes I made in this public widget.